int32[4] rgb_values
int32 color_temp
float32 lux
